src/MAlias.lua

```diff
@@ -165,6 +165,7 @@ function M.resolve(self, name)
          if (isFile(fn)) then
             local a   = {}
             a[#a + 1] = "LD_LIBRARY_PATH=\"".. (LMOD_LD_LIBRARY_PATH or "") .. "\""
+            a[#a + 1] = "LD_PRELOAD=\"\""
```

Comment:
@wpoely86 This is probably also what @rtmclay is referring to, to ensure that the existing value of `$LD_PRELOAD` doesn't get ignored?